testing/libarchive-qt/APKBUILD

```diff
@@ -0,0 +1,25 @@
+pkgname=libarchive-qt
+pkgver=1.0.2
+pkgrel=0
+pkgdesc="A Qt based archiving solution with libarchive backend"
+arch="all"
+url="https://gitlab.com/marcusbritanicus/libarchive-qt"
+license="LGPL-3.0"
```

Comment:
LGPL-3.0 is not a valid SPDX license identifier, like GPL-2.0 it also needs a `-or-later` or `-only` suffix.